topics/xamarin/index.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases: mono
+created_by: Alexandru Stefan Popescu
+display_name: Xamarin
+github_url: https://github.com/xamarin
+logo: xamarin.png
+released: May 16 2011
+short_description: The Xamarin platform that allow you to develop applications for iOS and Android.
+topic: xamarin
+url: https://www.xamarin.com/
+wikipedia_url: https://en.wikipedia.org/wiki/Xamarin
+---
+The Xamarin platform consists of a number of elements that allow you to develop applications for iOS and Android:
```

Comment:
Could you condense this down a bit to just 1-3 sentences?

Reply:
Changed it. I'd like to use something that's already out there and not to create some sentences that may miss something or not cover everything.